roles/dtc/common/templates/ndfc_attach_vrfs_loopbacks.j2

```diff
@@ -0,0 +1,18 @@
+[
```

Comment:
Seeing a couple of issues here with develop merged in

- `attach_group` key needs to be updated to be `vrf_attach_group`
- I see the following error when I don't have any vrf loopback data defined

```bash
ansible.errors.AnsibleUndefinedVariable: 'dict object' has no attribute 'loopback_id'. 'dict object' has no attribute 'loopback_id'
failed: [nac-ndfc1 -> localhost] (item={'name': 'NaC-VRF04', 'vrf_id': 150004, 'vlan_id': 2004, 'vrf_attach_group': 'leaf3'}) => {
    "ansible_loop_var": "vrf",
    "changed": false,
    "msg": "AnsibleUndefinedVariable: 'dict object' has no attribute 'loopback_id'. 'dict object' has no attribute 'loopback_id'",
    "vrf": {
        "name": "NaC-VRF04",
        "vlan_id": 2004,
        "vrf_attach_group": "leaf3",
        "vrf_id": 150004
    }
}
```

Reply:
hey man, gonna look at refactoring this loopback attach stuff today anyway to try and get rid of the loop per vrf, will sort the above while im there.

Reply:
this has been resolved now too